### PR TITLE
remove sdg sv overrides

### PR DESCRIPTION
**Pull request by n-h-diaz**

### CODE REVIEW

File: tools/nl/titles/demo_svs_1.3k_names.csv [hunk 1]
```diff
@@ -2177,112 +2177,6 @@ Employed Population Below International Poverty Line With Gender = Male,dc/topic
 "Employed Population Below International Poverty Line With Age = 25 Years Old And Over, Gender",dc/topic/sdgSIPOVEMP1_age-SDGAgeEnumYGE25_gender,,Employed Population Aged 25 or Over Below International Poverty Line by Gender,,Employed Population Aged 25 or Over Below International Poverty Line by Gender
```
Comment (pradh) [inline tools/nl/titles/demo_svs_1.3k_names.csv:2177]: What about dc/topic stuff, should we drop?

Comment (n-h-diaz) [reply tools/nl/titles/demo_svs_1.3k_names.csv]: dropped topics too!

**Comment by jehangiramjad** [comment]
Hmm. Did you follow this readme to drop these dcids from the Google Sheets? once you do that and follow the steps, it will automatically update this file and also the other json file which will be used by the frontend.

https://github.com/datacommonsorg/website/blob/master/tools/nl/titles/README.md

**Comment by n-h-diaz** [comment]
> Hmm. Did you follow this readme to drop these dcids from the Google Sheets? once you do that and follow the steps, it will automatically update this file and also the other json file which will be used by the frontend.
> 
> https://github.com/datacommonsorg/website/blob/master/tools/nl/titles/README.md

thank you!! I ran the script to update the json (also thank you for the help with the tests!) :)

**Review by jehangiramjad — APPROVED** [review]
Thanks! Just make sure that the diffs in the integration_tests/test_data are all only for the chart titles of the SDG SVs. Looks like that to me.

**Comment by pradh** [comment]
Thanks!

**Review by pradh — APPROVED** [review]

**Pull request merged**